src/selfplay/tournament.cc

```diff
@@ -352,41 +352,39 @@ void SelfPlayTournament::PlayOneGame(int game_number) {
   auto player2_threads = player_options_[1][color_idx[1]].Get<int>(kThreadsId);
   game.Play(player1_threads, player2_threads, kTraining, syzygy_tb_.get(),
             enable_resign);
-  
-  // If game was aborted, it's still undecided.
-  if (game.GetGameResult() != GameResult::UNDECIDED) {
```

Comment:
Is there any potential we should handle a case where the game is 'actually' aborted like the old comment?

Reply:
Something like `if (!abort_ || game.GetGameResult() != GameResult::UNDECIDED) {`?
Or just `if (!abort_) {`?